### PR TITLE
Fix NullRef in ClientIpTelemetryInitializer

### PR DESCRIPTION
**Pull request by abaranch**
#242

### CODE REVIEW

File: test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs [hunk 1]
```diff
@@ -20,7 +16,7 @@ public static HttpContextAccessor CreateHttpContextAccessor(RequestTelemetry req
             var request = new DefaultHttpContext().Request;
             request.Method = "GET";
             request.Path = new PathString("/Test");
-            var contextAccessor = new HttpContextAccessor() { HttpContext = request.HttpContext };
+            var contextAccessor = new HttpContextAccessor { HttpContext = request.HttpContext };
```
Comment (dnduffy) [inline test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs]: Hmm, I kind of like the parenthesis ( ), when did those stop being needed?

File: test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs [hunk 2]
```diff
@@ -41,5 +37,24 @@ public static HttpContextAccessor CreateHttpContextAccessor(RequestTelemetry req
 
             return contextAccessor;
         }
+
+        public static HttpContextAccessor CreateHttpContextAccessorWithoutRequest(HttpContextStub httpContextStub, RequestTelemetry requestTelemetry = null)
+        {
+            var services = new ServiceCollection();
+
```
Comment (dnduffy) [inline test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs]: nitpick: please remove some of the extra blank lines.

File: test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs (hunk 1 above)
Comment (abaranch) [inline test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs]: You do not need them when you define properties inline.

---
In reply to: [70717232](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/244#discussion_r70717232) [](ancestors = 70717232)

**Comment by dnduffy** [comment]
:shipit:

File: test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs (hunk 2 above)
Comment (abaranch) [inline test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs]: There are no several blank lines one after the other.... Do you specifically not like line 44?..

---
In reply to: [70718145](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/244#discussion_r70718145) [](ancestors = 70718145)

**Pull request merged**